README.md

```diff
@@ -22,4 +22,29 @@ The root ACL token will be printed at the end of the cluster boot.
 to stop the cluster run:
 ```
 ./cluster-down
-```
\ No newline at end of file
+```
+
+#### Running a dev build of consul in the containers
+<details>
+  <summary>Click to expand!</summary>
+  
+* Build the appropriate binary with `GOOS=linux make dev` (or whatever OS version your underlying image is based upon)
```

Comment:
I would remove the `or whatever OS version your underlying image is based upon` because the underlying image used in the repo is linux based and it will always be, at least for now.
```suggestion
* Build the appropriate binary with `GOOS=linux make dev`
```